action.yml

```diff
@@ -0,0 +1,91 @@
+name: 'silverbackltd-build-action'
+author: 'Silverback Ltd.'
+description: 'A GitHub Action to build docker images for a silverback repository and push to a container registry'
+branding:
+  icon: 'check-circle'
+  color: 'green'
+
+inputs:
+  push:
+    description: 'Push the image to ghcr'
+    required: false
+    default: 'false'
+
+outpus:
+  built_images:
+    description: 'List of Docker images that were built and pushed.'
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
```

Comment:
Don't think you need this, suggest to put into docs as something the user will do anyways

Reply:
Also is fetch depth 0 required?

Reply:
this is not needed, hold over from an old repository